modules/n1ql/pages/n1ql-language-reference/typefun.adoc

```diff
@@ -1,36 +1,225 @@
 = Type Functions
-:description: Type functions perform operations that check or convert expressions.
+:description: The following checks if the supplied expression is a (data type).
 :page-topic-type: concept
 
 [abstract]
 {description}
 
+NOTE: If any arguments to any of the following functions are [.out]`MISSING` then the result is also [.out]`MISSING` (i.e.
+no result is returned).
+Similarly, if any of the arguments passed to the functions are `NULL` or are of the wrong type (e.g.
+an integer instead of a string), then `NULL` is returned as the result.
+
 == Type-Checking Functions
 
-ISARRAY(expression)
+== ISARRAY(expression)
+
+=== Description
```

Comment:
Needs separate Description, Arguments, and Return Value sections, as per functions template

```suggestion
=== Description

Checks if the supplied expression is an array.

=== Arguments

expression:: [Required] The expression to check.

=== Return Value
```